RecoTracker/FinalTrackSelectors/python/preDuplicateMergingGeneralTracks_cfi.py

```diff
@@ -18,7 +18,7 @@
         cms.InputTag("muonSeededTracksInOutSelector","MVAVals"), 
         cms.InputTag("muonSeededTracksOutInSelector","MVAVals"), 
     ),
-    setsToMerge = cms.VPSet(cms.PSet(pQual = cms.bool(False), tLists = cms.vint32(0, 1,2))),
+    setsToMerge = cms.VPSet(cms.PSet(pQual = cms.bool(True), tLists = cms.vint32(0, 1,2))),
```

Comment:
this is the only (intentional) change in behavior.
needed to set confirmed for the muon-seeded (that are the only one we care, after all!)